### ISSUE TITLE
multipass launch ubuntu:xenial should work

### ISSUE THREAD

**Issue by dpb1** [canonical/multipass#210]
Would be nice if ubuntu[]/:]xenial works as it does in lxc launch ubuntu:xenial.

I realize that multipass is focused on Ubuntu thus the 'ubuntu' is not necessary, but if I optionally specify it, it would be nice to recognize it.

**Comment by townsend2010** [comment]
Yeah, maybe allow those aliases would help for users of lxc/lxd.  However, I'm not really a fan of allowing the `/` since lxc/lxd doesn't allow that either.

@Saviq, any opinions?

**Comment by Saviq** [comment]
We'll solve this by allowing you to configure the remote names and aliases as you wish. We'll stick to the colon notation though for separating remote and alias.
